src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue32316.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue32316 : _IssuesUITest
+{
+
+	public Issue32316(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "RTL mode: Padding for the label is not mirroring properly";
+
+	[Test]
+	[Category(UITestCategories.Label)]
+	public void RTLModePaddingShouldWork()
+	{
+		App.WaitForElement("ToggleFlowDirectionButton");
+		App.Tap("ToggleFlowDirectionButton");
+		VerifyScreenshot();
```

Comment:
Pending snapshots, running a build.

Reply:
Snapshots are available in the latest build.
<img width="789" height="563" alt="image" src="https://github.com/user-attachments/assets/79232aa9-f0cb-4ed3-8a7e-a1dafd4f2338" />
Could you commit the images?